src/DDTrace/Integrations/Memcached/MemcachedIntegration.php

```diff
@@ -125,7 +125,9 @@ function (SpanData $span, $args, $retval) use ($integration, $command) {
                 }
                 if (!is_array($args[0])) {
                     $integration->setServerTags($span, $this);
-                    $span->meta['memcached.query'] = $command . ' ' . Obfuscation::toObfuscatedString($args[0]);
+                    $queryParams = dd_trace_env_config("DD_TRACE_MEMCACHED_OBFUSCATION") ?
+                        Obfuscation::toObfuscatedString($args[0]) : $args[0];
+                    $span->meta['memcached.query'] = $command . ' ' . $queryParams;
```

Comment:
My feeling is that since all the changes have the same structure, we _could_ do a standard function for this. At least, if we ever have an issue, this would prevent us from having to look at five different places. This could be considered a nit :-)